app/renderer/components/Inspector/SelectedElement.js

```diff
@@ -83,11 +83,18 @@ export default class SelectedElement extends Component {
       });
     }
 
+    const showInteractionsLoadingIndicators = !elementInteractionsNotAvailable && !elementId;
```

Comment:
this condition is not very clear to me

Reply:
Yeah it was unnecessary to have it outside of the JSX, fix pushed.